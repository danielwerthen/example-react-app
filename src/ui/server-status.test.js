import React from 'react';
import ReactDOM from 'react-dom';
import ServerStatus from './server-status';

describe('ServerStatus', () => {
  it('should match snapshot', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ServerStatus />, div);
    expect(div.innerHTML).toMatchSnapshot();
  });
  it('should match snapshot when running', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ServerStatus
      running
    />, div);
    expect(div.innerHTML).toMatchSnapshot();
  });
});
