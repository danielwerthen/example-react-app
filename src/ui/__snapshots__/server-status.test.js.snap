exports[`ServerStatus should match snapshot 1`] = `"<div data-reactroot=\"\" class=\"server-status\"><p></p><p>Stopped</p><p></p><dl><dt>port</dt><dd></dd><dt>protocol</dt><dd></dd></dl><a>Edit</a><span> </span><a href=\"/undefined/start\">Start</a></div>"`;

exports[`ServerStatus should match snapshot when running 1`] = `"<div data-reactroot=\"\" class=\"server-status\"><p></p><p>Running</p><p></p><dl><dt>port</dt><dd></dd><dt>protocol</dt><dd></dd></dl><a>Edit</a><span> </span><a href=\"/undefined/stop\">Stop</a></div>"`;
